src/mo_frontend/printers.ml

```diff
@@ -5,16 +5,14 @@ open Parser.MenhirInterpreter
 
 let abstract abs con = abs
 
```

Comment:
I think this should be:
```ocaml
let abstract abs con = (abs, con)
```
Then string of symbols will have type `symbol -> (AbstractString, ConcreteString)`, so `string_of_symbol` and `exmaple_of_symbol` should be the same function. I think that was intention of the `abstract` function.